SUMMARY.md

```diff
@@ -0,0 +1,28 @@
+## Table of Contents
+
+* [Ternary Operator](content/TernaryOperator/TernaryOperator.q.md) ** - minutes**
+    > The ternary operator is used as a shorthand way of doing `if/else` statements. In this lesson, you will learn how to use the ternary operator to perform shorthand comparisons in PHP.
+
+* [Type Hinting](content/TypeHinting/TypeHinting.q.md) ** - minutes**
+    > Discover how you can use type hinting to define the data type that is passed in a function declaration.
+
+* [Cookies](content/Cookies/Cookies.q.md) ** - minutes**
+    > Learn how to create, retrieve, modify, and delete cookies using the `setCookie()` function.
+ 
+* [Sessions](content/Sessions/Sessions.q.md) ** - minutes**
+    > Learn how to start, get, modify, and destroy sessions using the `session_start()` function.
```

Comment:
I think we need to remove the phrase __using the `session_start()` function__ since a session is different from a cookie. Please try to consider the following in here...
```
// Starting session
session_start();
 
// Destroying session
session_destroy();

//storing and accessing session data
$_SESSION[]
```

Or you can rephrase the above statement. Thanks.